Godeps/Godeps.json

```diff
@@ -843,12 +888,17 @@
 			"Rev": "a98ad7ee00ec53921f08832bc06ecf7fd600e6a1"
 		},
 		{
-			"ImportPath": "github.com/vjeantet/asn1-ber",
-			"Rev": "85041cd0f4769ebf4a5ae600b1e921e630d6aff0"
+			"ImportPath": "github.com/vjeantet/goldap/message",
+			"Rev": "c4f081986193996a259a69268965dd16aca3d730"
 		},
 		{
 			"ImportPath": "github.com/vjeantet/ldapserver",
-			"Rev": "19fbc46ed12348d5122812c8303fb82e49b6c25d"
+			"Comment": "v1.0-10-g3a8d560",
```

Comment:
Why did this change? It's not used upstream that I'm aware of? Seems like something weird happened with your godep save